targettests/execution/complex_vector_synthesis.cpp

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// clang-format off
+// RUN: nvq++ %cpp_std --enable-mlir %s -o %t  && %t | FileCheck %s
+// TODO-FIX-KERNEL-EXEC
```

Comment:
What does this mean? And why not fix it here?

Reply:
Just copied from another test - I believe it was an ask from Ben to add the comment and an issue to update all the tests after we switch to new launch kernel for good. So it will be fixed after:)

Reply:
Found Ben's ask: https://github.com/NVIDIA/cuda-quantum/pull/2094#discussion_r1727102688
It is currently a cleanup item in issue https://github.com/NVIDIA/cuda-quantum/issues/1931

Reply:
OK. We're tracking it somewhere which is sufficient.